ow_lander/scripts/camera_capture_action_server.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+
+import actionlib
+
+import ow_lander.msg
+from std_msgs.msg import Empty, Float64
+from sensor_msgs.msg import PointCloud2
+
+class CameraCaptureActionServer(object):
+  #_result = std_msgs.msg.Bool()
+
+  def __init__(self, name):
+    self._action_name = name
+    self._result = ow_lander.msg.CameraCaptureResult()
```

Comment:
Is the result message actually a good initializer, as opposed to None?  Might it be stale data? (I don't know the details of action message management).

Reply:
I don't understand the question. http://wiki.ros.org/actionlib_tutorials/Tutorials/Writing%20a%20Simple%20Action%20Server%20using%20the%20Execute%20Callback%20%28Python%29#CA-3786eb3a74acabd7ea4ba08b8b23ad9583d5bd79_12 shows _result initialized at the class level, outside the __init__ method. With my limited Python experience, I don't know what the difference is between these two styles.

Reply:
Class level defined variables are static data members, where as those defined in __init__ or any other member method are non-static data members specific to the instance of the class (`self`). 

Given how there are never more than 1 instance of any of our action servers, the difference between one or the other is meaningless.

Reply:
I don't know Python or ROS well enough to make a substantial claim here, so you can ignore my first comment :)

Reply:
Actually, that was bugging me a little while making this: How should an action server handle multiple simultaneous requests? I didn't see any good examples of it, but that's probably a problem for another day.